deployments/data102/image/environment.yml

```diff
@@ -0,0 +1,44 @@
+name: data102-FA23
+
+channels:
+- conda-forge
+
+dependencies:
+- python==3.11.0
+- git==2.39.1
+- jupyter-resource-usage==1.0.0
+- jupyterlab==4.0.4
+- jupyterlab-favorites==3.0.0
+- jupyterlab_server==2.23.0
+- jupyterlab_widgets==3.0.8
+- jupyter_server==2.7.0
+- nbgitpuller==1.1.1
+- notebook==7.0.2
+- ipywidgets==8.0.7
+- jinja2==3.1.2
+- jupyterlab==4.0.4
```

Comment:
jupyterlab is present twice, lines 10 and 19. No need to change this PR, but perhaps on the next one.

Reply:
tbh, i'd prefer to just remove it now...

Reply:
To the conda resolver it doesn't matter. If you don't want to have to sit through CI again, it can wait. But if there's no time pressure it can be removed now.